python/lsst/analysis/tools/interfaces/_interfaces.py

```diff
@@ -103,18 +103,23 @@ def __init__(self) -> None:
 like an NDArray should be considered a Vector.
 """
 
-KeyedData = MutableMapping[str, Vector | Scalar | HealSparseMap | Tensor]
-"""KeyedData is an interface where either a `Vector` or `Scalar` can be
-retrieved using a key which is of str type.
+KeyedData = MutableMapping[str, Vector | Scalar | HealSparseMap | Tensor | Mapping[str, Any]]
```

Comment:
Does `Mapping` need `[str, Any]` here?

Reply:
(and elsewhere below)

Reply:
I was under the impression that `Mapping` needs to specify the key and value types. I can double-check.

Reply:
Tested it without using `[str, Any]` and there were no issues.